### DISCUSSION TITLE
Different Fonts per session

### DISCUSSION THREAD

**Discussion by nelgin** [Eugeny/tabby#6672] [General]
I opened an issue on this, but I figure here is a more appropriate place to ask.

I connect to many different servers and have different font requirements for each. How to I configure fonts per session? I only see the ability to configure them globally. I guess I missed something or it's a programming oversight?

Thanks,

**Comment by Eugeny** [comment]
There is no way to do this yet.

**Reply by nelgin** [reply]
Any idea as to when it may be implemented? Unfortunately the software is no good to me without it.
